### PR TITLE
Add aboe/chrony to metadata, change /dev/shm to mount resource, move TMOUT, update deps

### PR DESCRIPTION
**Pull request by canihavethisone**

### CODE REVIEW

**Comment by canihavethisone** [comment]
PDK spec tests pass locally but many are failing in Travis - not sure what is going on there....

File: .fixtures.yml [hunk 1]
```diff
@@ -27,7 +27,6 @@ fixtures:
       ref: "1.9.0"
     augeas: "camptocamp/augeas"
     cron: "puppet/cron"
-    augeas_core: "puppetlabs/augeas_core"
```
Comment (bjvrielink) [inline .fixtures.yml]: This is needed for spec tests to pass with Puppet6.

Comment (canihavethisone) [reply .fixtures.yml]: thanks, added.  Do we want to be testing against P6 as this module (and probably many deps) show >6 ?

**Comment by bjvrielink** [comment]
> PDK spec tests pass locally but many are failing in Travis - not sure what is going on there....

PDK spec only tests for 1 Puppet version (most likely Puppe5.5). Travis does test with 2 different Puppet versions: Puppet5 and Puppet6.
Puppet did remove several modules from the core Puppet code when they released Puppet6, including augeas. In order to make augeas work on Puppet6, you need to use the augeas-core module from the forge.

File: .fixtures.yml (hunk 1 above)
Comment (bjvrielink) [reply .fixtures.yml]: I personally am in favor of supporting Puppet6 (thus updating the metadata), but only after gaining some confidence (thus the spec tests).

**Comment by canihavethisone** [comment]
I think it would be a good idea to lift the puppet version in metadata to >=5.0.0 <7.00, as most deps are >5 and this module is now 6 compliant.  Would you like to do it or I can push the updated metadata

**Comment by bryanjbelanger** [comment]
Thanks as alway @canihavethisone and @bjvrielink .

**Pull request merged**